src/main/java/org/fsg1/fmms/backend/services/CurriculaService.java

```diff
@@ -39,10 +41,14 @@ public String getQueryCurriculaString() {
      *
      * @return A JSON ObjectNode representing an array of curricula.
      */
-    public JsonNode get(final String query, final Object... parameters) throws SQLException, IOException {
+    public JsonNode get(final String query, final Object... parameters) throws SQLException, IOException,
+            EntityNotFoundException {
         final ResultSet resultSet = getConn().executeQuery(query, parameters);
```

Comment:
ResultSets needs to be closed after use.
Add try-with-resources.
See:
https://docs.oracle.com/javase/8/docs/api/java/sql/ResultSet.html#close--